.github/workflows/bootstrapping_ci.yml

```diff
@@ -14,18 +14,18 @@ on:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.sha || github.ref }}
-  cancel-in-progress: false
+  cancel-in-progress: true
```

Comment:
Imho, leave that to `false`.

Reply:
If it would fix something false would correct. Tho false is false here 😅. It should not be used. 

In the review comment above you mention a workflow.
> See [vlang/v/actions/runs/8890038625/attempts/1](https://github.com/vlang/v/actions/runs/8890038625/attempts/1) .
Both jobs got canceled.

The reason for the fail was that it was run scheduled. 
Checking run #66:
![Screenshot_20240501_101348](https://github.com/vlang/v/assets/34311583/5e0794cf-4608-477c-985d-6ea99e231988)
You removed scheduled runs. So this was solved there. If scheduled runs were kept, a update to concurrency group name for the master branch that considers this would be the right solution.  

The main reason why it should be false I mentioned in comment linked in the description:

> When pushing to master the concurrency group uses the sha of the commit, so there will be no cancelation anyway.
-> In other words, on master it is a separate concurrency group without a possible cancellation.
>
> Settings this to always false will only stack up the CI pipline with this workflow when there are repeated pushes to PRs. Then there will waiting for old irrelevant runs of this workflow to finish before the relevant of a PR starts.
Therefore it should stay true.

Reply:
i.e. you think that the <https://github.com/vlang/v/actions/runs/8890038625/attempts/1> cancelations, were caused by an older periodic/scheduled run?

Reply:
> Checking the linked run 66
I do not know where do you see that.
Can you please link to it?

Reply:
I see this:
![image](https://github.com/vlang/v/assets/26967/e0b757a8-68a2-45ee-aecb-b33ee98cfcb3)

Reply:
> i.e. you think that the [vlang/v/actions/runs/8890038625/attempts/1](https://github.com/vlang/v/actions/runs/8890038625/attempts/1) cancelations, were caused by an older periodic/scheduled run?

Based on the available data I'm confident that is the reason.

> Can you please link to it?

Sure, here a link to the overview of the bootstrapping_ci runs on master:
https://github.com/vlang/v/actions/workflows/bootstrapping_ci.yml?query=branch%3Amaster
When scrolling down a bit you find:
![Screenshot_20240501_134752](https://github.com/vlang/v/assets/34311583/1b7693be-c3bd-4397-af06-8c22c11ecdd7)

The overview shows the triggers next to the workflow name. In the summary of a single workflow it's not immediately visible.